### ISSUE TITLE
Image support with preview 

### ISSUE THREAD

**Issue by Terkyz** [jpkhawam/Nabu#47]
I know Nabu is still in alpha, and a lot of things still need to be added, but it would be great if they added image support!

**Comment by jpkhawam** [comment]
I agree, the reason it's not a priority is I don't know how to go about it yet. Notes are just plain text in EditTexts. I tried checking Markdown a while back but it seemed like a hassle. It's definitely a welcome feature though so thanks for opening an issue
